_posts/2023-07-24-single-vs-multi-tenant.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: postv2
+font: serif
+title: "Single Tenant vs. Multi-Tenant - Unveiling the Ideal Architecture For Your Business"
+description: "How to choose the right architecture for your business."
+date: 2023-07-31
+highlight: monokai
+image: /img/single-multi-tenant/single_multi_tenant.png
+author: Shermayne Lee
+tags: [single tenant, multi-tenant, solution architect]
+---
+
+Hey there, tech enthusiasts and curious minds, welcome to our blog post on the ongoing debate between single tenant and multi-tenant architectures!
```

Comment:
I do not like using "single" and "multi" when describing tenancy.  They are confusing.  
Does multi-tenancy mean that there is a single instance deployed with multiple clients within it or does it mean there are multiple instances deployed with a single client within each?  Easy to get the context wrong and misunderstand.
For this reason, using isolated and shared tenancy as we do on WayFinder (or siloed and pooled tenancy as AWS WAR does) is much clearer.
I would suggest discussing this within this post and using that discussion for your choice of terminology.

Reply:
This is a valid point. 

Tho, the reason im using single vs multi describing tenancy as the terminology is because that is the common language used in the industry. I want ppl to be able to relate on what it means in those common term . Honestly in my opinion ,  as long as your instance are shared with more than one client that it is a multi tenant . You can do one instance to multiple client , one instance to a group of multiple clients, multiple instance to multiple client , multiple instance to one client